Sources/App/Features/Push/Controllers/PushController.swift

```diff
@@ -43,4 +44,25 @@ struct PushController: RouteCollection {
 
         return .noContent
     }
+
+    private func sendNotification(req: Request) async throws -> HTTPStatus {
+        struct SendNotificationRequest: Decodable {
+            let message: String
+            let securityCode: String
+        }
+
+        let notificationRequest = try req.content.decode(SendNotificationRequest.self)
+        let secretSecurityCode = Environment.get("PUSH_SECURITY_CODE")
+
+        guard notificationRequest.securityCode == secretSecurityCode else { throw Abort(.forbidden) }
+
+        let tokens = try await Token.query(on: req.db).all()
+        let alert = APNSwiftAlert(title: "SwiftLeeds", body: notificationRequest.message)
+
+        tokens.forEach { token in
+            _ = req.apns.send(alert, to: token.token)
+        }
```

Comment:
This feels like functionality which won't scale too great with more devices/users. 

Shouldn't necessarily be on this pull request, but have we considered setting up [Queues](https://docs.vapor.codes/advanced/queues/) to allow us to offload this step to a different process?

Reply:
I believe this will scale for now, but if we start to got 1000+ devices then we'd probably need a better system.  Other options are to remove users that fail the push send to reduce the number we send out

Reply:
Yeah not one that needs to be over-engineered right now but a queue would add some safety, scaling, and ensure that these endpoints are responding quickly.

That could become more important depending on the use case of this API. 

If we need to hit this API multiple times in a relatively short time then it could start to cause issues.

Reply:
Yeah, this is a stepping stone.  Looking forward to seeing a more optimised solution in the future.